### DISCUSSION TITLE
Postgis Unable to Query

### DISCUSSION THREAD

**Discussion by yassinGIS** [giswqs/geog-312#60] [Q&A] [answered]
Hi, I created a postgis after watching your video from other course. I then loaded the database with a table that had a very large number of records (72M). So, I need to query the table before I can use it in geopandas for further processing. However, I am not able to run the WHERE clause in my query. It keeps telling me that the column I am querying doesn't exist. I tried to change the syntax and everything but it won't run the query. I had the same issue running the query in QGIS. Arcgis couldn't read the table.

**Comment by giswqs** [comment, marked as answer]
Sorry, I no longer use PostGIS. You might want to check out DuckDB. 
https://geog-414.gishub.org/book/duckdb/04_data_import.html
